src/types/lib/types.vcxproj

```diff
@@ -22,6 +22,7 @@
     <ClCompile Include="..\MenuEvent.cpp" />
     <ClCompile Include="..\ModifierKeyState.cpp" />
     <ClCompile Include="..\ScreenInfoUiaProviderBase.cpp" />
+    <ClCompile Include="..\sgrStack.cpp" />

```

Comment:
we'll need to slap this into the `sources` or `sources.inc` file in this folder as well!